frontend/app/component/form/data_entry/state/dataEntryUtils.ts

```diff
@@ -329,12 +215,12 @@ export function getInitialFormState(election: Required<Election>, defaultFormSta
       acceptWarnings: false,
       hasChanges: false,
       acceptWarningsError: false,
-      errors: [],
-      warnings: [],
+      errors: new ValidationResultSet(),
+      warnings: new ValidationResultSet(),
     };
   });
 
-  return structuredClone({ ...result, ...defaultFormState });
```

Comment:
`structuredClone` does not preserve classes, so I had to remove it. Because the `result` object is already reinitialised in every function call this should not matter.

The `defaultFormState` was not used so I removed it.